<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GM-Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressource.qrc">
    <normaloff>:/data/images/GMA.png</normaloff>:/data/images/GMA.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox1">
        <property name="title">
         <string>Scenario</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout1">
         <item>
          <widget class="QCustomTreeWidget" name="treeScenario">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QGroupBox" name="groupBox2">
        <property name="title">
         <string>History</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout2">
         <item>
          <widget class="QCustomTreeWidget" name="treeHistory">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QGroupBox" name="groupBox4">
        <property name="title">
         <string>Notes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout4">
         <item>
          <widget class="QTextEdit" name="textNotes"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QGroupBox" name="groupBox5">
        <property name="title">
         <string>Characters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout5">
         <item>
          <widget class="QCustomTableWidget" name="tableWidget"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QGroupBox" name="groupBox6">
        <property name="title">
         <string>Sound effects</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout6">
         <item>
          <widget class="QCustomTreeWidget" name="treeFX">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QGroupBox" name="groupBox3">
        <property name="title">
         <string>Music</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout3">
         <item>
          <widget class="QCustomTreeWidget" name="treeMusic">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout1">
           <item>
            <widget class="QPushButton" name="buttonMusic">
             <property name="text">
              <string>&amp;Play</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderMusic">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
    <addaction name="actionS_ave_as"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create a new game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="statusTip">
    <string>Load a game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="statusTip">
    <string>Save the current game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionS_ave_as">
   <property name="text">
    <string>S&amp;ave as</string>
   </property>
   <property name="statusTip">
    <string>Save the current game in a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>QCustomTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>QCustomTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ressource.qrc"/>
 </resources>
 <connections/>
</ui>
